### ISSUE TITLE
Profesion inscripcion

### ISSUE THREAD

**Issue by lilith384** [RoDNet/Bugtracker-Infinium-4.0.6a#1318]
Me he estado subiendo inscripcion hasta rasganorte y cuando me salia investigar rasganorte lo tenia en rojo y no me subia puntos de inscripcion y aparte cuando investigaba no me daba  para hacer cosas nuevas o amarillas o rojas solo verdes y apenas suben la profesion.

**Comment by ghost** [comment]
Solo vengo a decir que faltan glifos, tal vez sea porque usaste (creo) la base de datos 4.3.4 redneo, para poner los glifos que faltaban, pero en realidad ahora faltan glifos que antes si estaban.

Seguro por eso illith384 no puede subir inscripción, porque le faltan algunos glifos que se podían obtener del instructor de inscripción, pero ahora ya no.

**Comment by redneo** [comment]
No, creo recordar que habia un problema con ciertos spells que te tendria que subir un punto de habilidad y no lo hacen, no estoy al 100% seguro...

El tema de los glifos, solo meti los que faltaban en los libros, los que habia ya, siguen estando.

Los colores de las profesiones como sabeis, para que suba skill es: 

Rojo 100%
Amarillo 75%
Verde 25%
Gris 0%

**Comment by ghost** [comment]
@redneo ah, pero de todas formas faltan glifos que te da el instructor, como por ejemplo "glifo de socorro oscuro", ahora ningun inscriptor lo tiene, pero por ejemplo yo lo tengo aprendido con mi DK.

Si quieres abro otro issue.

**Comment by redneo** [comment]
Ese glifo, no lo venden ni en el server normal ni en el test, a saber como lo has conseguido ¬¬ ahora ese SI que lo venden, si sabes de más, dimelo.

entrenan*
